.gitignore

```diff
@@ -4,6 +4,7 @@
 target
 config
 src/main/ui/node_modules
+src/main/ui/package-lock.json
```

Comment:
If you look at the link I included it says:

> This file is intended to be committed into source repositories

Therefore we should include it!